tests/operator/operator.go

```diff
@@ -3281,7 +3281,7 @@ func prometheusRuleEnabled() bool {
 	prometheusRuleEnabled, err := util.IsPrometheusRuleEnabled(virtClient)
 	if err != nil {
 		fmt.Printf("ERROR: Can't verify PrometheusRule CRD %v\n", err)
-		panic(err)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
I'd rather replace the entire `if` with the `Expect` here. IIRC you should use ExpectWithOffset to modify the call stack since this is inside a `func`.